views/includes/page-templates/beneficial-owner-individual.html

```diff
@@ -4,17 +4,26 @@
   <div class="govuk-grid-column-two-thirds">
     {% include "includes/list/errors.html" %}
 
+    {% if relevant_period %}
+      {% include "includes/page-templates/important-banner.html" %}
+    {% endif %}
+
     <span class="govuk-caption-xl">Beneficial owner</span>
 
     <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
-    <br> <p class="govuk-body">You can add more later.</p> <br>
-
-    {% if entity_number %}
-      {% include "includes/ceased-date-details-summary-new-bo.html" %}
+    {% if relevant_period %}
+      {{ govukInsetText({
+        text: 'You need to enter information that is correct as at the date they ceased to be a registrable beneficial owner.'
```

Comment:
string enclosed here in single quotes. Changing these to double quotes is more readable and consistent with majority of text: strings in html files. e.g. 
'You need to enter information that is correct as at the date they ceased to be a registrable beneficial owner.'
->
"You need to enter information that is correct as at the date they ceased to be a registrable beneficial owner."

Reply:
Thanks, I've updated this :)